contracts/interfaces/IOnChainVault.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+struct StrategyParams {
+    uint256 performanceFee;
+    uint256 activation;
+    uint256 debtRatio;
+    uint256 lastReport;
+    uint256 totalDebt;
+    uint256 totalGain;
+    uint256 totalLoss;
+}
+
+
+interface IOnChainVault {
```

Comment:
The name is not clear, 'cause all of the vaults are onchain =) Suggestion to rename to: IVault? Or ICrosschainVault?